Directory.Build.props

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>8.1.5</VersionPrefix>
+    <VersionPrefix>8.1.6</VersionPrefix>
     <VersionSuffix>preview-03</VersionSuffix>
```

Comment:
I think the version should be 8.1.5, and the only change would be updating the VersionSuffix to preview-04

Reply:
I said the same hehehehe....

Reply:
And please remember to update the CHANGELOG.md file as well :)